src/pages/complete-profile/complete-profile.tsx

```diff
@@ -0,0 +1,130 @@
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+
+import 'open-props';
+import '../../index.css';
+
+const root = document.getElementById('root') as HTMLDivElement;
+
+createRoot(root).render(
+	(
+		<StrictMode>
+			<form action='' encType='multipart/form-data'>
+				<h2>Complete your profile</h2>
+
+				<details open>
+					<summary>In a nutshell:</summary>
+
+					<div>
+						<label htmlFor='name'>Name</label>
+						<input id='name' name='name' type='text' />
+					</div>
+					<div>
+						<label htmlFor='email'>
+							E-mail
+							<span>REQUIRED</span>
+						</label>
+						<input id='email' name='email' type='email' required />
+					</div>
+					<div>
+						<label htmlFor='slack-handle'>
+							Slack handle
+							<span>REQUIRED</span>
+						</label>
+						<input
+							id='slack-handle'
+							name='slack-handle'
+							type='text'
+							aria-description='Your slack handle to TorontoJS'
+							placeholder='Your slack handle to TorontoJS'
+						/>
+					</div>
+					<div>
+						<label htmlFor='pronouns'>Pronouns</label>
+						<input id='pronouns' name='pronouns' type='text' list='pronouns-options' placeholder='Your pronouns (optional)' />
+						<datalist id='pronouns-options'>
+							<option>He/him</option>
+							<option>She/her</option>
+							<option>They/them</option>
+						</datalist>
+					</div>
+					<div>
+						<span>Date of birth</span>
+						<br />
+						<label htmlFor='month'>Month</label>
+						<select id='month' name='month'>
```

Comment:
We will need  Data selector component. 

https://github.com/torontojs/volunteer/issues/31

Reply:
For this specific situation the regular `<input type="date" />` or with type of `month` does not work, as it includes the year.

But yeah, this can be turned into a component.